mapie/regression.py

```diff
@@ -698,17 +711,17 @@ def predict(
                 )
                 for _alpha in alpha_np
             ]
-        ).data
+        )
         y_pred_up = np.column_stack(
             [
                 np_nanquantile(
                     upper_bounds.astype(float),
-                    1 - _alpha,
+                    1-_alpha,
```

Comment:
wring linting, there should be a space before and after `-`

Reply:
Done.